packages/web/components/Carousel/index.tsx

```diff
@@ -32,6 +50,14 @@ export const Carousel: React.FC<{ gap: number; children: JSX.Element[] }> = ({
 
   const [isGreaterThanLg] = useMediaQuery(`(min-width: ${breakpoints.lg})`);
 
+  const isGreaterThanBase = useBreakpointValue({ base: true, md: false });
+
+  const isGreaterThanMd = useBreakpointValue({ base: false, md: true });
+
+  const isGreaterThanXl = useBreakpointValue({ base: false, xl: true });
+
+  const isGreaterThan2Xl = useBreakpointValue({ base: false, '2xl': true });
+
   useEffect(() => {
```

Comment:
Seems like this could be a `useMemo` and be a bit cleaner, where it returns `itemWidth`, `constraint` and potentially `multiplier`

Reply:
Thank you. ✅